src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -835,9 +835,18 @@ private function addValidationSection(ArrayNodeDefinition $rootNode)
                                 ->end()
                             ->end()
                         ->end()
-                        ->booleanNode('disable_not_compromised_password')
-                            ->defaultFalse()
-                            ->info('Disable NotCompromisedPassword Validator: the value will always be valid.')
+                        ->arrayNode('not_compromised_password')
+                            ->addDefaultsIfNotSet()
+                            ->children()
+                                ->booleanNode('enabled')
+                                    ->defaultTrue()
+                                    ->info('When disabled, compromised passwords will be accepted as valid.')
+                                ->end()
+                                ->scalarNode('endpoint')
+                                    ->defaultValue('https://api.pwnedpasswords.com/range/%s')
```

Comment:
I'm wondering if we shouldn't leave the default to the implementation, which means defaulting to `null` here.

Reply:
Sounds good, with the new constant and nullable parameter this would work nicely. I'd just mention the actual default in the following doc PR.